lib/geofencing.dart

```diff
@@ -2,8 +2,164 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-export 'package:geofencing/src/geofencing.dart'
-    hide geofenceEventToInt, intToGeofenceEvent;
```

Comment:
Is there any reason this was removed and the other source files moved? The purpose of this file is to hide internal methods.

Reply:
Fixed. I had missed the directory structure here.